### ISSUE TITLE
ConvertTo-WikiTable | ConvertTo-WikiStorageFormat issue

### ISSUE THREAD

**Issue by jdubbbb** [brianbunke/ConfluencePS#1]
It seems like ConvertTo-WikiTable should return a string instead of an array.

`Get-Service | select name,status -First 3 | ConvertTo-WikiTable | ConvertTo-WikiStorageFormat`

I get multiple tables for each row:

```
<table><tbody>
<tr>
<th><p>Name</p></th>
<th><p>Status</p></th>
</tr>
</tbody></table>

<table><tbody>
<tr>
<td><p>AdobeARMservice</p></td>
<td><p>Running</p></td>
</tr>
</tbody></table>

<table><tbody>
<tr>
<td><p>AeLookupSvc</p></td>
<td><p>Stopped</p></td>
</tr>
</tbody></table>

<table><tbody>
<tr>
<td><p>ALG</p></td>
<td><p>Stopped</p></td>
</tr>
</tbody></table>
```

I tried a lot of combinations to try to get one table but finally figured out out-string fixes it:

`Get-Service | select name,status -First 3 | ConvertTo-WikiTable | out-string | ConvertTo-WikiStorageFormat`

```
<table><tbody>
<tr>
<th><p>Name</p></th>
<th><p>Status</p></th>
</tr>
<tr>
<td><p>AdobeARMservice</p></td>
<td><p>Running</p></td>
</tr>
<tr>
<td><p>AeLookupSvc</p></td>
<td><p>Stopped</p></td>
</tr>
<tr>
<td><p>ALG</p></td>
<td><p>Stopped</p></td>
</tr>
</tbody></table>
```

Or maybe ConvertTo-WikiStorage should convert it to string. Or at least somewhere in the help documentation mentioning Out-String for ConvertTo-WikiTable.

**Comment by brianbunke** [comment]
Hey, thanks! Those are helpful examples. I'm at VMworld right now, but will definitely review next week.

**Comment by brianbunke** [comment]
Resolved in a8bef873f02c215f82387f6449d78039b4d01120

Decided to `Out-String` the output of `ConvertTo-WikiTable`. The previous output was an array of row strings, which wasn't intended. This way also still allows multiple tables to be piped into `ConvertTo-WikiStorageFormat` at once, if desired.

Added an example to address this common use case, which I had already referenced in the description.
